backend/components/rasa/src/main/java/co/airy/core/rasa_connector/MessageHandler.java

```diff
@@ -40,50 +41,34 @@ public Message getMessage(Message contactMessage, MessageSendResponse response)
                 .build();
     }
 
+   
+
     public String getContent(String source, MessageSendResponse response) throws JsonProcessingException {
         final String text = response.getText();
-        if (text == null) {
+        final String image = response.getImage();
+
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode rootNode = mapper.createObjectNode();
+        ObjectNode childNode1 = mapper.createObjectNode();
+        ObjectNode childNode2 = mapper.createObjectNode();
+
+        if (text == null && image == null) {
```

Comment:
Added image support for the Rasa connector: the Rasa assistant can send both text & image
<img width="359" alt="Screenshot 2022-11-18 at 15 35 47" src="https://user-images.githubusercontent.com/38159391/202731590-59180209-6b0e-488b-afb6-46e195c4300f.png">